src/server/administer/index.md

```diff
@@ -20,6 +20,9 @@ If you do not want to provide these data, you can opt-out any time by setting th
 COLLECT_STATISTICS=false
 ```
 
+<ServerType type="EE" />
 ::: warning
-It is legally required by <EnterprisePlatformName /> servers to send usage information to monitor the usage of the License. 
+It is legally required by <EnterprisePlatformName /> servers to send usage information (`call-home` service) to monitor the usage of the License. 
+
+This service runs on background via `TCP` port `443`, hence it needs to be enabled on firewall side in order to comply with <EnterprisePlatformName /> terms.
```

Comment:
This should not be part of the warning - warning is about legal bit - port is description how to set it up

Reply:
In terms of the technical requirement, it should say that it *requires outgoing https access to https://api.merginmaps.com/monitoring on TCP port 443*

I think this page has some wider issues though - it looks like it's written as a call to action to disable monitoring (which we don't want). I would instead recommend:

- changing the title so it describes monitoring, not how to disable it
- update the example to show monitoring enabled
- detail the technical requirements above
- say what it's for (making mm better and required for monitoring of enterprise edition licence usage)
- describe how to disable it if desired in the community edition while noting that this is now allowed in EE